README.markdown

```diff
@@ -99,6 +100,18 @@ STP relies on : boost, flex, bison and minisat. You can install these by:
 $ sudo apt-get install cmake bison flex libboost-all-dev python perl minisat
 ```
 
+If your distribution does not come with minisat, STP maintains an updated fork; it can be built as follows:
```

Comment:
I would rather use periods instead of semicolons in both instances in this change; use whichever you feel is appropriate.

Reply:
I have no preference, I'm happy for `.` over `;`.

Reply:
Done!